guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,23 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select *Schedule Remote Job*.
+. In the *Job category* field, select `Convert to RHEL`.
+. In the *Job template* field, verify that the Job Template is set to `Convert to RHEL`.
+. On the *Job Invocation* page, define the main job settings.
++
+The *Search Query* field displays the name of any search you have performed, or the list of hosts you have selected.
++
+. In the *Activation Key* field, enter the host activation key.
+. Optional: Click *Show Advanced Fields* if you want to configure further aspects.
+. Click *Restart*.
```

Comment:
Where did this come from? Restart is Yes/No select box. I guess you meant Click *Submit*?